am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/FilterResourceResponse.java

```diff
@@ -12,6 +12,6 @@
 @Builder
 public class FilterResourceResponse {
     private String resourceId;
-    private JsonNode data;
+    private Map<String, JsonNode> data;
```

Comment:
That seems wrong. We want to return JsonNode but without properties that consumer should not see.

Reply:
👍